tests/e2e/hcp_cluster_test.go

```diff
@@ -0,0 +1,177 @@
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			var err error
+			isHosted, err = clusterService.IsHostedCPCluster(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+
+		})
+
+		AfterEach(func() {
+			By("Clean the cluster")
+			rosaClient.CleanResources(clusterID)
+		})
+
+		It("create and edit hosted-cp cluster with AuditLog Forwarding enabled/disabled via rosacli - [id:64491]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Get cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+
+				if !isHosted || clusterDetail.AuditLogForwarding != "Enabled" {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+				role := clusterDetail.AuditLogRoleARN
+				Expect(role).ToNot(BeEmpty())
+
+				By("Edit the cluster to disable audit log forwarding")
+				_, err = clusterService.EditCluster(
+					clusterID,
+					"--audit-log-arn", "",
+					"-y",
+				)
+				Expect(err).To(BeNil())
+
+				By("Get cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+
+				By("Edit the cluster to enable audit log forwarding")
+				_, err = clusterService.EditCluster(
+					clusterID,
+					"--audit-log-arn", role,
+					"-y",
+				)
+				Expect(err).To(BeNil())
+
+				By("Get cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+				Expect(clusterDetail.AuditLogForwarding).To(Equal("Enabled"))
+				Expect(role).To(Equal(role))
+
+			})
+
+		It("create cluster with the KMS and etcd encryption  for hypershift clusters by rosa-cli - [id:60083]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Check the help message of 'rosa create cluster -h'")
+				output, err := clusterService.RetrieveHelpForClusterCreate()
+				Expect(err).To(BeNil())
+				Expect(output.String()).To(ContainSubstring("--kms-key-arn"))
+				Expect(output.String()).To(ContainSubstring("--etcd-encryption"))
+				Expect(output.String()).To(ContainSubstring("--enable-customer-managed-key"))
+
+				By("Get cluster description")
+				jsonData, err := clusterService.GetJSONClusterDescription(clusterID)
+				Expect(err).To(BeNil())
+
+				if !isHosted {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+
+				ectdKMS := jsonData.DigString("aws", "etcd_encryption", "kms_key_arn")
+				npKMS := jsonData.DigString("aws", "kms_key_arn")
+
+				if jsonData.DigBool("aws", "etcd_encryption") {
+					Expect(ectdKMS).ToNot(BeEmpty())
+				}
+				Expect(jsonData.DigString("aws", "kms_key_arn")).To(Equal(npKMS))
```

Comment:
I don't think so. We are using different keys in the day1 preparation. They shouldn't be expected to be the same.